### PR TITLE
chore: remove unused cluster and capture modules, simplify lib/index.js

### PR DESCRIPTION
**Pull request by kyegupov**
- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?

Apparently, the tricky code in `snyk()` function in `lib/index.js` is not used anymore, so it can be just dropped, paving the way for proper conversion of all of the `lib` to Typescript.

### CODE REVIEW

**Comment by kyegupov** [comment]
Kick flaky AppVeyor/Travis builds

File: src/lib/index.js [hunk 1]
```diff
@@ -1,44 +1,12 @@
...
-        'and API token');
-    }
-    // hook();
-    require('./capture')();
-  }
-
-  return snyk;
-}
+var snyk = {};
+module.exports = snyk;
```
[the remark this answers is not in the document]
Comment (darscan) [reply src/lib/index.js]: > this patch removes that 
or, at least, changes the signature in a breaking way

[the remark this answers is not in the document]
Comment (darscan) [reply src/lib/index.js]: in reality though.. I have no idea what's going on here..

[the remark this answers is not in the document]
Comment (kyegupov) [reply src/lib/index.js]: @darscan this only breaks importing snyk as a function and calling it.
Does anybody do that? Do we want to allow that?
My guess is "no" on both counts. Whatever this function does, this doesn't make any sense anymore.
Who can give a more informed answer on that?

**Pull request merged**